chapters/extend/executor.rst

```diff
@@ -166,7 +202,7 @@ Each :class:`Executor` has a core method, which defines the algorithmic behavior
 | :class:`BaseEvaluator`  |   :meth:`evaluate`          |
 +-------------------------+-----------------------------+
 
-Feel free to override other methods/properties as you need. But frankly, most of the extension can be done by simply overriding the core methods listed above. Nothing more. You can read the source code of our executors for details.
+Feel free to override other methods/properties as you need. But frankly, most of the extension can be done by simply overriding the core methods listed above.
```

Comment:
`frankly`  seems a strange word in a technical document...
`But frankly,` -> `But probably,`